taichi/gui/gui.h

```diff
@@ -825,13 +833,21 @@ class GUI : public GUIBase {
             "to exit gracefully)");
       }
     }
-    process_event();
     while (last_frame_interval.size() > 30) {
       last_frame_interval.erase(last_frame_interval.begin());
     }
     auto real_fps = last_frame_interval.size() /
                     (std::accumulate(last_frame_interval.begin(),
                                      last_frame_interval.end(), 0.0_f));
+    if (show_gui) {
```

Comment:
In fact, we can ignore the FPS-saving `sleep` procedure when we just want to save images.

Reply:
@archibate actually should I also move `taichi::Time::wait_until(last_frame_time + frame_delta_limit)` to under the `if (show_gui)`?

Reply:
Yes, I think actually we can move all the codes to `update_gui()`? Or simply add `if (!show_gui) return;` to the first line of `update()`?